conf/application.conf

```diff
@@ -18,6 +18,7 @@ play.modules.enabled += "inject.ApplicationProductionModule"
 # ~~~~~
 application.settingsWidget.enabled = true
 #application.countries = ["DE", "US", "GB", "AT", "CH"] # uncomment to override countries from CTP project, default first
+application.currencies = ["EUR"] # uncomment to override currencies from CTP project, default first
```

Comment:
Yes, because I can't get the currencies from the Project (missing field in `Project` class), so I have no other option than to use them from the config file.